all-actors/src/main/java/org/sunbird/JsonKey.java

```diff
@@ -58,7 +58,7 @@
     String PUBLIC_KEY = "publicKey";
     String DESCRIPTION = "description";
     String LOGO = "logo";
-    String ISSUE_DATE = "issueDate";
+    String ISSUE_DATE = "issuedDate";
```

Comment:
Are you sure this will not affect the API?

Reply:
yes , actually in the request it is issuedDate , not issueDate,